docs/HowToGuides/GettingStarted.md

```diff
@@ -259,10 +260,14 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
        --sccache --release-debuginfo --swift-disable-dead-stripping --test \
        --xcode
      ```
+   Linux (uses Ninja):
+     ```sh
+     utils/build-script --release-debuginfo --test --skip-early-swift-driver
+     ```
    This will create a directory
    `swift-project/build/Ninja-RelWithDebInfoAssert`
    (with `Xcode` instead of `Ninja` if you used `--xcode`)
-   containing the build artifacts.
+   containing the Swift compiler, clang/LLVM, and standard library build artifacts.
```

Comment:
```suggestion
   containing the build artifacts for the Swift compiler, the Swift standard
   library, Clang and LLVM.
```
Having LLVM in-between two Swift-related projects seems a bit odd order-wise.

Reply:
Changed the order.